eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -273,10 +287,19 @@ private static Optional<Dependency> artifact(final Path file) throws IOException
      * @return Coords
      */
     private static String coords(final Dependency dep) {
-        return String.format(
-            "%s:%s:%s",
-            dep.getGroupId(), dep.getArtifactId(), dep.getVersion()
-        );
+        String ret = "";
+        if (dep.getClassifier().isEmpty()) {
+            ret = String.format(
+                "%s:%s:%s",
+                dep.getGroupId(), dep.getArtifactId(), dep.getVersion()
+            );
+        } else {
+            ret = String.format(
+                "%s:%s:%s:%s",
+                dep.getGroupId(), dep.getArtifactId(), dep.getClassifier(), dep.getVersion()
+            );
+        }
```

Comment:
@MikhailLipanin Can we simplify this part? Just by calculating the `classifier` and pass it to `String.format`